scripts/setup_linux.sh

```
@@ -0,0 +1,165 @@
+  $SUDO apt-get install -y --no-install-recommends \
+    binutils clang cmake llvm lld ninja-build pkg-config \
+    libgtk-3-dev libsecret-1-0 libsecret-1-dev libunwind-dev \
+    libasound2-dev libgstreamer1.0-dev \
+    libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev \
+    libmpv-dev mpv wget \
+    tesseract-ocr
+
+  # Install Playwright system deps
+  log "Installing Playwright system dependencies..."
+  if [[ -d ".venv" ]]; then
+    source .venv/bin/activate 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Not following: .venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)